source/server.js

```diff
@@ -332,18 +332,18 @@ app.get('/api/fs/exists', ensureAuthenticated, (req, res) => {
 });
 
 app.get('/api/fs/listDirectories', ensureAuthenticated, (req, res) => {
-  const dir = req.query.term.trim();
+  const dir = path.resolve(req.query.term.trim()).replace("/~", "");
```

Comment:
Wouldn't it be better to check if it starts with `/~` and replace it with [`os.homedir()`](https://nodejs.org/api/os.html#os_os_homedir).
Then this would also work on windows.

Example:
```js
var dir = req.query.term.trim();
if (dir.startsWith("/~")) {
    dir = path.join(os.homedir(), dir.substring(2));
}
```

Reply:
Well, for linux it should work with "~".  My thought behind using `path.resolve` is that it has more considerations around path resolving then I can think of and thus more trust worthier